src/vaishg10695/HomeWork19_01_2020.java

```diff
@@ -0,0 +1,53 @@
+/*String s = "213hs.[][$%&ABF";
+		//1. Capitals letters			//2. Digit
+										//.a Odd Number sum
+		//3. Special characters			//4. Even Number sum 	//5. Small characters
+		
+		//Output
+		//ABF4.[][$%&2hs*/
+
+package vaishg10695;
+
+public class HomeWork19_01_2020 {
+
+	static void displayString(String s) {
+		String capitalLet = "";
+		String specialChar = "";
+		String smallLet = "";
+		int oddSum = 0;
+		int evenSum = 0;
+
+		for (int i = 0; i < s.length(); i++) {
+			String ch = "";
+
+			if (Character.isUpperCase(s.charAt(i))) {
+				capitalLet = capitalLet + s.charAt(i);
+			} else if (Character.isLowerCase(s.charAt(i))) {
+				smallLet = smallLet + s.charAt(i);
+			} else if (Character.isDigit(s.charAt(i))) {
+				ch = ch + s.charAt(i);
+				if (Integer.parseInt(ch) % 2 == 0)
+					evenSum = evenSum + Integer.parseInt(ch);
```

Comment:
Here You should write
Integer.valueOf(String.valueOf(ch));

this will give expecte output: only Integer.parseInt() gives you ascii value of char